.github/workflows/tls_cxx__build.yml

```diff
@@ -23,6 +23,8 @@ jobs:
         with:
           submodules: recursive
       - name: Build ${{ matrix.test.app }} with IDF-${{ matrix.idf_ver }}
+        env:
+          EXPECTED_WARNING: "unknown kconfig symbol"
```

Comment:
Would it be possible to make definition of `EXPECTED_WARNING` conditional? It is expected warning for IDF versions < 6